java/arcs/sdk/android/dev/service/AndroidRuntimeSettings.java

```diff
@@ -58,10 +58,11 @@
   private static final boolean DEFAULT_USE_CACHE_MANAGER = true;
   // Disables system trace
   private static final String DEFAULT_SYSTEM_TRACE_CHANNEL = "";
-  // Disables worker pool
-  private static final boolean DEFAULT_USE_WORKER_POOL = false;
-  // Uses the default settings (no additional configurations)
-  private static final String DEFAULT_WORKER_POOL_OPTIONS = "";
+  // Activates worker pool
+  private static final boolean DEFAULT_USE_WORKER_POOL = true;
+  // "nosuspend" prepares or replenishes new workers ahead of time but never
+  // re-uses (suspend-then-resume) them.
```

Comment:
Could you add a comment why this is beneficial? Is the benefit of 'keeping it clean from side-channels' a security benefit? Are there any potential downsides to setting this option?

Reply:
It's not a security benefit; instead, it keeps the same security level as that the current design is witht the 'nosuspend' option.

Without 'nosuspend' option, we get the most performant configuration which manages workers, suspends and resurrects them on demand to serve new coming Arcs, whereas with 'nosuspend', we only spin up new workers ahead of time in pool to get ready for subsequent Arcs.

A worker's global scope (window or global or self in web) is isolated and dedicated for one Arc's particle execution context at current design. If we re-use the same worker via suspend/resume to serve new coming Arcs, it ends up allowing new Arcs to read old Arcs' data if the old Arcs didn't clean their data i.e.
old Arc:
window.myData = "you can't see me";
new Arc using the same worker:
console.log(`eavesdropping ${window.myData}`);

Reply:
Added more explanation to where interfaces are defined. (I usually put all options' details there).